apps/web/src/pages/signin.tsx

```diff
@@ -31,33 +25,10 @@ export default function Signin() {
           <Flex
             justifyContent="center"
             marginTop="32px"
-            marginBottom="48px"
             flexDirection={'column'}
           >
-            <Input
-              placeholder="Email"
-              marginBottom="20px"
-              value={loginForm.email}
-              onChange={(e) =>
-                setLoginForm({ ...loginForm, email: e.target.value })
-              }
-            />
-            <Input
-              type="password"
-              placeholder="Password"
-              marginBottom="20px"
-              value={loginForm.password}
-              onChange={(e) =>
-                setLoginForm({ ...loginForm, password: e.target.value })
-              }
-            />
-            <Button
-              background="#4C658A"
-              color="#FFF"
-              onClick={async () => login(loginForm.email, loginForm.password)}
-            >
-              Sign In
-            </Button>
+            <SignIn />
```

Comment:
I like that you separated the html out into components here, makes it a lot more organized and matches how the other pages are structured